nx/lib/nx/serving.ex

```diff
@@ -19,7 +19,7 @@ defmodule Nx.Serving do
         import Nx.Defn
 
         defnp print_and_multiply(x) do
-          print_value({:debug, x})
+          print_value(x, label: "debug")
```

Comment:
@josevalim there's still a problem with using `print_value` here.

When I tried it on my machine, it printed nothing.

Given [its docs](https://hexdocs.pm/nx/Nx.Defn.Kernel.html#print_value/2):

```
The return value of this function must be part of the output
```

I think that's because it's not the last expression inside the `print_and_multiply` function.